lighthouse-core/lib/i18n/i18n.js

```diff
@@ -160,10 +160,11 @@ function _formatIcuMessage(locale, icuMessageId, icuMessage, values) {
   // fallback to the original english message if we couldn't find a message in the specified locale
   // better to have an english message than no message at all, in some number cases it won't even matter
   const messageForMessageFormat = localeMessage || icuMessage;
+  if (messageForMessageFormat === undefined) throw new Error('No ICU message string to format');
   // when using accented english, force the use of a different locale for number formatting
   const localeForMessageFormat = locale === 'en-XA' ? 'de-DE' : locale;
   // pre-process values for the message format like KB and milliseconds
-  const valuesForMessageFormat = _preprocessMessageValues(icuMessage, values);
+  const valuesForMessageFormat = _preprocessMessageValues(messageForMessageFormat, values);
```

Comment:
@patrickhulce note this change.. it was using the fallback message instead of the one pulled from the locales files.  which seemed odd.  right?